data/new-product-direction/test.txt

```diff
@@ -0,0 +1,7 @@
+gruntwork-io/terraform-aws-security
```

Comment:
NIT: Is `test.txt` just for testing or still needed?

Reply:
@brikis98 we've decided to move all of these data scripts to `prototypes` so disregard this one and yes it shouldn't exist.